src/stores/search/searchActions.ts

```diff
@@ -1,16 +1,48 @@
 import { Dispatch } from 'redux';
 
-import { SearchOptionsModel } from '../../models';
+import { PAGE_SIZE } from '@api/recipes.api';
```

Comment:
page size shouldn't be a part of api layer, 'cause you are using it on redux layer. 
api layer should only receive params and not define constants